common/tasks/trigger-jenkins-job/0.1/README.md

```diff
@@ -0,0 +1,3 @@
+# Tekton Task: Trigger Jenkins Job
+
+Please read/update [trigger-jenkins-job tasks docs](../../../../docs/qe-available-tasks/trigger-jenkins-job.md) from docs folder.
```

Comment:
Can you add an description to the .md you reference here on how this task is used?. More users that have konflux will benefit from this